frontend/src/components/Search.tsx

```diff
@@ -1,67 +1,63 @@
-import { TiDeleteOutline } from 'react-icons/ti'
-import '../css/Search.css'
-import { useState, useContext } from 'react'
-import { GlobalContext } from '../context/GlobalState'
+import { TiDeleteOutline } from "react-icons/ti"
+import "../css/Search.css"
+import { useState, useContext } from "react"
+import { GlobalContext } from "../context/GlobalState"
 
 //////
 
 // import React, { useMemo, useContext } from 'react'
-import { useLoadScript } from '@react-google-maps/api'
+import { useLoadScript } from "@react-google-maps/api"
 import usePlacesAutocomplete, {
   getGeocode,
   getLatLng,
-} from 'use-places-autocomplete'
+} from "use-places-autocomplete"
 
 import {
   Combobox,
   ComboboxInput,
   ComboboxPopover,
   ComboboxList,
   ComboboxOption,
-} from '@reach/combobox'
-import '@reach/combobox/styles.css'
-import { LatLng } from './Homepage'
+} from "@reach/combobox"
+import "@reach/combobox/styles.css"
+import { LatLng } from "./Homepage"
 
 /////
 
 const Search = () => {
   const { handleSearch, setCenter, setRadius } = useContext(GlobalContext)
 
   // eslint-disable-next-line

```

Comment:
You can remove this line, I think it won't complain. If it does, then it's likely you need to add <string> type after useState

eg `useState<string>(...)`